includes/definitions.inc.php

```diff
@@ -1019,7 +1019,7 @@
 $config['os'][$os]['icon']             = 'avaya';
 
 $os = 'avaya-vsp';
-$config['os'][$os]['text']             = 'Avaya VSP';
+$config['os'][$os]['text']             = 'VOSS';
```

Comment:
How come this is changed to VOSS? It's only a text description and Avaya VSP seems more relevant still.